lib/src/main/java/io/ably/lib/http/TokenAuth.java

```diff
@@ -36,13 +36,13 @@ public void setTokenDetails(TokenDetails tokenDetails) {
 		this.encodedToken = Base64Coder.encodeString(tokenDetails.token).replace("=", "");
 	}
 
-	public TokenDetails authorise(AuthOptions options, TokenParams params) throws AblyException {
+	public boolean authorise(AuthOptions options, TokenParams params, boolean force) throws AblyException {
```

Comment:
I am not sure I follow why `force` is added as an argument here when `force` is an attribute of the `options` being passed in and can be determined using the logic in `authorise`, https://github.com/ably/ably-java/blob/0a0f3ecb1d0810d26fceb4e2f9b80614649ed642/lib/src/main/java/io/ably/lib/rest/Auth.java#L420